README.md

```diff
@@ -156,19 +156,19 @@ complete(myEntity).ok().done();
 The following is an example
 
 ```
-var sdk = require('kinvey-backend-sdk');
-var service = sdk.service(function(err, service) {
-  var dataLink = service.dataLink;   // gets the datalink object from the service
-  var notImplementedHandler = function(request, complete) {
+let sdk = require('kinvey-backend-sdk');
+let service = sdk.service((err, service) => {
+  let dataLink = service.dataLink;   // gets the datalink object from the service
+  let notImplementedHandler = (request, complete) => {
```

Comment:
These should be `const`.  Also the line re the `notImplementedHandler` should be removed.